cranelift-codegen/src/stack_layout.rs

```diff
@@ -103,7 +114,10 @@ pub fn layout_stack(frame: &mut StackSlots, alignment: StackSize) -> CodegenResu
     offset = offset
         .checked_sub(outgoing_max)
         .ok_or(CodegenError::ImplLimitExceeded)?;
-    offset &= -(alignment as StackOffset);
+
+    if must_align {
```

Comment:
```suggestion
    if frame.values().next().is_some() {
```

Reply:
This will align the stack if there are incoming arguments only, which seems unnecessary.

Reply:
Didn't notice that `IncomingArg` doesn't have `must_align = true`.